response_operations_ui/controllers/collection_instrument_controllers.py

```diff
@@ -44,14 +44,16 @@ def upload_collection_instrument(collection_exercise_id, file, form_type=None):
             form_type=form_type,
             status=response.status_code,
         )
-        return False
+        if response.headers["Content-Type"] == "application/json":
```

Comment:
Is this code like this to handle 400 vs 500 errors?  If so, can you put a comment next to it as it's definitely different to how we normally do it.

Reply:
I pulled it from the one below upload_ru_specific_collection_instrument, its very debatable if we need it, but we did have a test that didn't provide json, perhaps its that which needs updating